lib/modules/dosomething/dosomething_api/resources/campaign_resource.inc

```diff
@@ -274,14 +274,42 @@ function _campaign_resource_signup($nid, $values) {
   if (!isset($values['source'])) {
     $values['source'] = NULL;
   }
+  // @TODO: handle passing this to Rogue so it is not lost (and add to Phoenix documentation which is missing this)
   if (!isset($values['transactionals'])) {
     $values['transactionals'] = TRUE;
```

Comment:
I'm not sure, it's not in the Phoenix docs so I didn't realize it existed until I moved from Rogue to the Phoenix side